maintainers/README.md

```diff
@@ -43,7 +43,8 @@ The team meets twice a week:
 - Discussion meeting: [Fridays 13:00-14:00 CET](https://calendar.google.com/calendar/event?eid=MHNtOGVuNWtrZXNpZHR2bW1sM3QyN2ZjaGNfMjAyMjExMjVUMTIwMDAwWiBiOW81MmZvYnFqYWs4b3E4bGZraGczdDBxZ0Bn)
 
   1. Triage issues and pull requests from the [No Status](#no-status) column (30 min)
-  2. Discuss issues and pull requests from the [To discuss](#to-discuss) column (30 min)
+  2. Discuss issues and pull requests from the [To discuss](#to-discuss) column (30 min).
+     Once a month, this slot is used to check the [Assigned](#assigned) column to make sure that nothing bitrots in it.
```

Comment:
By whom? What's the actual process?

The purpose of assigning work is to make an individual responsible, so that _not_ the whole team needs to put effort into the issue.

```suggestion
     Once a month, each team member checks the [Assigned](#assigned) column for prs/issues assigned to them, to either
       - unblock it by providing input
       - mark it as draft if it is blocked on the contributor
       - escalate it back to the team by moving it to To discuss, and leaving a comment as to why the issue needs to be discussed again.
```

Reply:
A team member - not any individual.

> to check the column together was to redistribute/re-negotiate assignments

That's what the third option is for. Why should anything that's _assigned_ be bubbled back up to the team without the _assignee_ even looking at it beforehand? That'd be a waste of meeting time.